src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working ;( ");

```

Comment:
Similarly, the message "Truck stopped working ;( " should be more informative. Including the class name would make it clear which type of machine has stopped working.

Reply:
but I added the class name in the message